lib/fog/azurerm/docs/network.md

```diff
@@ -72,21 +77,22 @@ Get a single record of virtual network
 Get virtual network object from the get method and then destroy that virtual network.
 
 ```ruby
-      vnet.destroy
+    vnet.destroy
 ```
 
 ## Create Subnet
 
 Create a new Subnet
 
 ```ruby
-    subnet = azure_network_service.subnets.create(
-        name: '<Subnet name>',
-        resource_group: '<Resource Group name>',
-        virtual_network_name: '<Virtual Network name>',
-        address_prefix: '10.1.0.0/24'
-    )   
- )
+    subnet = network.subnets.create(
+      name: '<Subnet Name>',
+      resource_group: '<Resource Group Name>',
+      virtual_network_name: '<Virtual Network Name>',
+      address_prefix: '10.0.0.0/24',
+      network_security_group_id: '/subscriptions/######/resourceGroups/fog-rg/providers/Microsoft.Network/networkSecurityGroups/fog-nsg',
```

Comment:
use place holders for subscription-id, resource-group and network-security-group